flow/project/commands/deploy_contracts/deploy.go

```diff
@@ -148,6 +148,7 @@ var Cmd = &cobra.Command{
 		if len(errs) == 0 {
 			fmt.Println("\n✅ All contracts deployed successfully")
 		} else {
+			// REF: better output when errors
```

Comment:
Good idea. This could even be an issue linked in the code:

```suggestion
			// TODO: better output when errors: ISSUE_LINK
```